openedx/core/djangoapps/oauth_dispatch/docs/decisions/0007-include-organization-in-tokens.rst

```diff
@@ -0,0 +1,192 @@
+7. Include Organization in Tokens
```

Comment:
@bill-filler @georgebabey @douglashall Can at least one of you review this particular file?  We can also meet to discuss in person as necessary.  Thanks.